test-data/unit/check-expressions.test

```diff
@@ -1330,6 +1330,16 @@ def f(t: Tuple[int, ...]) -> None:
     '%d %d %d' % t
 [builtins fixtures/primitives.pyi]
 
+[case testStringInterpolationUnionType]
+from typing import Tuple, Union
+a: Union[Tuple[int, str], Tuple[str,int]] = ('A', 1)
+'%s %s' % a
+'%s' % a  # E: Not all arguments converted during string formatting
+
+b: Union[Tuple[int, str], Tuple[int, int], Tuple[str, int]] = ('A', 1)
+'%s %s' % b
+'%s %s %s' % b  # E: Not enough arguments for format string
```

Comment:
Add test case where one union item is valid and another is invalid. It should be an error if any item is invalid.

Reply:
done ✅